content/html/concepts/elements/elements.md

```diff
@@ -12,12 +12,18 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-An HTML element is a piece of content in an HTML document and uses the following syntax:
+An HTML element is a piece of content in an HTML document.
 
-```html
-<opening tag> content <closing tag></closing></opening>
+## Syntax
+
+HTML elements typically use the following syntax:
+
+```pseudo
+<opening tag>content</closing tag>
```

Comment:
```suggestion
<opening tag> content </closing tag>
```

Adding spacing around 'content' makes it easier to read in file.

Reply:
This suggestion does not change anything in regards to rendering so I am not accepting it.